src/flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.cc

```diff
@@ -0,0 +1,299 @@
+      {"CRTC_Y", &property_ids->plane.crtc_y},
+      {"CRTC_W", &property_ids->plane.crtc_w},
+      {"CRTC_H", &property_ids->plane.crtc_h},
+      {"FB_ID", &property_ids->plane.fb_id},
+      {"CRTC_ID", &property_ids->plane.crtc_id},
+      {"CRTC_ID", &property_ids->connector.crtc_id},
+  };
+  struct drm_property_address connector_table[] = {
+      {"CRTC_ID", &property_ids->connector.crtc_id},
+  };
+
+  GetPropertyAddress(drm_crtc_->crtc_id, DRM_MODE_OBJECT_CRTC, crtc_table,
+                     sizeof(crtc_table) / sizeof(drm_property_address));
+  GetPropertyAddress(drm_plane_id_, DRM_MODE_OBJECT_PLANE, plane_table,
+                     sizeof(plane_table) / sizeof(drm_property_address));
+  GetPropertyAddress(drm_connector_id_, DRM_MODE_OBJECT_CONNECTOR,
+                     connector_table,
+                     sizeof(connector_table) / sizeof(drm_property_address));
+}
+
+void NativeWindowDrmEglstream::GetPropertyAddress(
+    uint32_t id, uint32_t type,
+    NativeWindowDrmEglstream::drm_property_address* table, size_t length) {
+  auto properties = drmModeObjectGetProperties(drm_device_, id, type);
+  if (properties) {
+    for (uint32_t i = 0; i < properties->count_props; i++) {
+      auto property = drmModeGetProperty(drm_device_, properties->props[i]);
+      if (property) {
+        for (uint32_t j = 0; j < length; j++) {
+          if (strcmp(table[j].name, property->name) == 0) {
+            *(table[j].ptr) = property->prop_id;
+            break;
+          }
+        }
+      }
+      drmModeFreeProperty(property);
+    }
+    drmModeFreeObjectProperties(properties);
+  }
+}
+
+bool NativeWindowDrmEglstream::CreatePropertyBlob() {
+  if (drmModeCreatePropertyBlob(drm_device_, &drm_mode_info_,
+                                sizeof(drm_mode_info_),
+                                &drm_property_blob_) != 0) {
+    LINUXES_LOG(ERROR) << "Failed to create property blob";
+    return false;
+  }
+  return true;
+}
+
+bool NativeWindowDrmEglstream::CreateFb() {
+  struct drm_mode_create_dumb create_dump = {0};
+  create_dump.width = drm_mode_info_.hdisplay;
+  create_dump.height = drm_mode_info_.vdisplay;
+  create_dump.bpp = 32;
+  if (drmIoctl(drm_device_, DRM_IOCTL_MODE_CREATE_DUMB, &create_dump) != 0) {
+    LINUXES_LOG(ERROR) << "Failed to create dumb buffer";
+    return false;
+  }
+
+  if (drmModeAddFB(drm_device_, drm_mode_info_.hdisplay,
+                   drm_mode_info_.vdisplay, 24, 32, create_dump.pitch,
+                   create_dump.handle, &drm_fb_) != 0) {
+    LINUXES_LOG(ERROR) << "Failed to add a framebuffer";
+    return false;
+  }
+
+  struct drm_mode_map_dumb map_dump = {0};
+  map_dump.handle = create_dump.handle;
+  if (drmIoctl(drm_device_, DRM_IOCTL_MODE_MAP_DUMB, &map_dump) != 0) {
+    LINUXES_LOG(ERROR) << "Failed to map dumb buffer";
+    return false;
+  }
+
+  auto map =
+      static_cast<uint8_t*>(mmap(0, create_dump.size, PROT_READ | PROT_WRITE,
```

Comment:
There isn't unmap.

Reply:
I created https://github.com/sony/flutter-embedded-linux/issues/71 for this issue.